pkg/interface/src/logic/lib/publish.ts

```diff
@@ -0,0 +1,117 @@
+import { Post, GraphNode, TextContent, Graph, NodeMap } from "~/types";
+
+const buntPost = (): Post => ({
+  author: '',
+  contents: [],
+  hash: null,
+  index: '',
+  signatures: [],
+  'time-sent': 0
+});
```

Comment:
Perhaps these "generic post functions" could be moved into a generic `lib/post.ts` library.